performance-tests/bench_2/worker/WriteAction.cpp

```diff
@@ -15,6 +15,7 @@ WriteAction::WriteAction(ACE_Proactor& proactor)
  , max_count_(0)
  , new_key_count_(0)
  , new_key_probability_(0)
+ , final_wait_for_ack_{DDS::DURATION_INFINITE_SEC, DDS::DURATION_INFINITE_NSEC}
```

Comment:
So if the config file doesn't specify a value, it may wait forever on a call to `stop()`?